### ISSUE TITLE
Facebook Messenger does not install

### ISSUE THREAD

**Issue by gameme** [Android-L-Porting-Team/Android-L-Mako#139]
Facebook Messenger fails to install from play store with message: "Unknown Error Code during application Install: -112"

I know this is not a big deal, but this is like the only app that fails to install on my phone right now.

**Comment by jecordei** [comment]
Have you performed a factory reset prior to updating the ROM?

**Comment by gameme** [comment]
Oh yeah, clean wipe. I was actually stuck in recovery mode, so I had to re-flash the stock 4.4.4 before I re-did the whole installation process.

Does that mean it works for you?

**Comment by Sykopompos** [comment]
That is cause you are in enforcing mode and app install is borked in the aosp sepolicy base (aosp issue)

**Comment by gameme** [comment]
I am sorry I dont understand that. Is that suppose to be helping me fix ?

**Comment by aurangzaib** [comment]
It's working since first beta

**Comment by edwinfinch** [comment]
I've been able to install Facebook messenger just fine on beta 4 and up.

**Comment by gameme** [comment]
Good to know that, I wish that helped me fix the problem though :(

**Comment by edwinfinch** [comment]
Yeah sorry I wish I had something helpful, I just figured I'd add what my experience was. 

I wish WiFi tethering worked. I have to cry myself to sleep every night.

**Comment by edwinfinch** [comment]
But maybe one day all of the bugs will be squished and we can all go to sleep without crying.
